account_move_line_stock_info/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Account Move Line Stock Move",
+    "version": "8.0.1.0.0",
+    "depends": [
+        "stock_account"
+    ],
+    "author": "Eficent,"
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
```

Comment:
In the end it's more practical that people should get access to the OCA repo. We're not maintaing any page in our website that better describes this module.